src/ioc/ioc.react.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import { Container, interfaces } from 'inversify';
+import { useMemoOne } from 'use-memo-one';
+
+const InversifyContext = React.createContext<{ container: Container | null }>({ container: null });
+
+type Props = {
+    container: Container;
+};
+
+export const IoCProvider: React.FC<Props> = (props) => {
+    return (
```

Comment:
It's better to distructure props object. Smth like this: 
`export const IoCProvider: React.FC<Props> = ({ container, children }) => { }`